lib/std/Build.zig

```diff
@@ -597,22 +597,15 @@ pub const CreateModuleOptions = struct {
 /// `addModule` can be used instead to create a public module.
 pub fn createModule(b: *Build, options: CreateModuleOptions) *Module {
     const module = b.allocator.create(Module) catch @panic("OOM");
-    module.* = .{
-        .builder = b,
-        .source_file = options.source_file,
-        .dependencies = moduleDependenciesToArrayHashMap(b.allocator, options.dependencies),
-    };
+    module.* = Module.init(
+        b.allocator,
+        b,
+        options.source_file,
+        options.dependencies,
+    );
```

Comment:
Struct initialization is simpler since there is only one initialization site. I don't see any reason to change this.